pennylane/transforms/decompositions/single_qubit_unitary.py

```diff
@@ -43,100 +43,96 @@ def _convert_to_su2(U, return_global_phase=False):
     return (U_SU2, exp_angles) if return_global_phase else U_SU2
 
 
-def zyz_decomposition(U, wire):
```

Comment:
Don't feel obligated to do it because it is not required for unitary hack, but would it be possible to copy paste the previous function and create: `rot_decomposition with the keyword rot` because that what was zyz in fact?

Reply:
What exactly do I need to do? Copy paste the old function verbatim and add it to `supported_rotations` with key `rot`? Also, the original function didn't have support for returning global phase. I might also have to add tests for code coverage.

Which of the above do you want me to do?

Reply:
I added `_rot_decomposition` (accessible using `rot` in one_qubit_decomposition`) and ignored if `return_global_phase` was passed as `True`. Also added tests for it. Didn't add it as a supported option in documentation for `one_qubit_decomposition` though.

Reply:
Thanks that looks good to me!